docs/src/assets/styles/docsGlobal.scss

```diff
@@ -63,3 +63,8 @@ body,
     display: initial;
   }
 }
+
+ul {
+  // Fixed Firefox Issue of list style getting rendered as disc.
+  list-style: none;
```

Comment:
This will solve the issue but might open a few more problems.
Try to give CSS to specific element, for example, if it is applicable for sidebar only, give base class accordingly, so that it won't affect globally.